### PR TITLE
Basic routing

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Added Basic Routing. 

image

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks good to be merged

File: canvaplay/src/Pages/Watchlater/Watchlater.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (MaruthiPothuganti) [inline canvaplay/src/Pages/Watchlater/Watchlater.jsx:1]: you can exclude this as we dont need to import react in every file.

**Review by MaruthiPothuganti — APPROVED** [review]
The Structuring looks good. PR can be merged.

File: canvaplay/src/App.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
```
Comment (ashishkatiyar2404) [inline canvaplay/src/App.css:10]: you can write media queries in the last ....

**Review by ashishkatiyar2404 — APPROVED** [review]

File: canvaplay/src/Pages/Watchlater/Watchlater.jsx (hunk 1 above)
Comment (ShrutiLakhani) [reply canvaplay/src/Pages/Watchlater/Watchlater.jsx:1]: Sure. Thanks

**Pull request merged**